service/validation.go

```diff
@@ -48,7 +49,20 @@ func (v *parameterValidator) Validate(value interface{}) *ParameterWarning {
 		}
 		return v.newParameterWarning("required")
 	}
-
+	if v.parameter.Repeated {
+		// Check if the value is a slice
+		if reflect.TypeOf(value).Kind() != reflect.Slice {
+			return v.newParameterWarning("not an array")
+		}
+		array := reflect.ValueOf(value)
+		for i := 0; i < array.Len(); i++ {
```

Comment:
Tell me if I'm wrong, but here we can only repeated basic types, we can't repeat whole objects?

Reply:
there is no object structure for the moment, this will be another feature. So yes now only basic types because that the only one we have.

But when we manage the nested objects we can just make a recursive call.
`validateType` calls `validate`